src/ui/advanced/gestures.md

```diff
@@ -4,8 +4,29 @@ description: How gestures, such as taps and drags, work in Flutter.
 ---
 
 This document explains how to listen for, and respond to,
-_gestures_ in Flutter. Examples of gestures include
-taps, drags, and scaling.
+_gestures_ in Flutter.
+Examples of gestures include taps, drags, and scaling.
+
+You might have multiple gesture recognizers,
```

Comment:
I think I would move this new block under "Gesture disambiguation" to explain what the first sentence (about having multiple recognizers) means.